patches/lua-tree.patch

```diff
@@ -416,13 +414,7 @@ index 6c92098cc..c3c3c12fe 100644
  local pairs         = pairs
  local ipairs        = ipairs
 @@ -33,34 +26,12 @@ local fmt           = string.format
- local find          = string.find
- local gsub          = string.gsub
- local split         = pl_stringx.split
--local re_find       = ngx.re.find
--local re_match      = ngx.re.match
-+local re_find       = require "go.re2".find
-+local re_match      = require "go.re2".match
+ local re_match      = ngx.re.match
  local get_phase     = ngx.get_phase
  local ngx_sleep     = ngx.sleep
 -local inflate_gzip  = zlib.inflateGzip
```

Comment:
Did you test to make sure the patch applies correctly? I didn't so just a general question.

Reply:
I've verified the patch applies correctly.